QuoteVerification/QvE/Enclave/qve.cpp

```diff
@@ -1805,12 +1819,6 @@ quote3_error_t sgx_qve_verify_quote(
          p_quote_collateral->version != QVE_COLLATERAL_VERSION3 &&
          p_quote_collateral->version != QVE_COLLATERAL_VERSOIN31 &&
          p_quote_collateral->version != QVE_COLLATERAL_VERSION4) {
-#ifdef SERVTD_ATTEST
-		if(p_quote_collateral != NULL) {
```

Comment:
Looks like we should check the `collateral_flag`. If it is true, we need to call tdx_att_free_collateral() API to free the collateral resource we get from internal.  The comment also applies to L1813.

Reply:
Fixed.